src/Components/Components/src/Virtualization/Virtualize.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.AspNetCore.Components.Virtualization
+{
+    /// <summary>
+    /// Provides functionality for rendering a virtualized, fixed-size list of items.
+    /// </summary>
+    /// <typeparam name="TItem">The <c>context</c> type for the items being rendered.</typeparam>
+    public sealed class Virtualize<TItem> : VirtualizeBase<TItem>
+    {
+        protected override int ItemCount => Items.Count;
+
+        /// <summary>
+        /// Gets or sets the content template for list items.
+        /// </summary>
+        [Parameter]
+        public RenderFragment<TItem>? ChildContent { get; set; }
```

Comment:
In some components we support an alternative to `ChildContent` that has a name more obvious to the situation. That might be appropriate here too.

For example, if the developer was allowed to supply *either* `Item` or `ChildContent` (both of type `RenderFragment<TItem>`) then they could do either:

```razor
<Virtualize>
    <div>@context.Thing</div>
</Virtualize>
```

... or:


```razor
<Virtualize>
    <Item><div>@context.Thing</div></Item>
</Virtualize>
```

... which is a bit more natural if you're supplying multiple template parameters than calling it `ChildContent`.